workflows/responses.go

```diff
@@ -0,0 +1,8 @@
+package workflows
+
+type StateResponse struct {
+	WfInfo    WorkflowStruct
+	StartTime string `json:"start_time"`
```

Comment:
Why `string` and not `time.Time`?

Reply:
I was thinking that returning it as a string could provide more flexibility for formatting purposes or even flexibility for other backends if they return time as a string. If that doesn't make sense though, I'll gladly make the change to use time.Time

Reply:
Will there be any Go code that consumes this type? My concern is about whether 1) any Go code needs to consume this field and 2) whether it may be displayed somewhere. In the latter case, it would be a little awkward to have different components display the creation time using different formats. Not a big deal, but I wonder if it could cause problems, especially if we need to actually consume these values somewhere, like in the Dapr Dashboard. Having a strict type/format might also make it easier to display start times in a sorted manner.

Reply:
Updated the temporal code to output the time using a time standard (RFC3339). It is still a string